tests/unit/amplitude/expected_recipe.yml

```diff
@@ -28,10 +28,26 @@ Parameters:
   - Name: &par5 Phase_J/psi(1S)_to_f(0)(1500)_0+gamma_1;f(0)(1500)_to_pi0_0+pi0_0;
     Type: Phase
     Value: 0.0
-  - Name: MesonRadius_J/psi(1S)
+  - Name: &par6 MesonRadius_J/psi(1S)
     Type: MesonRadius
     Value: 1.0
     Fix: true
+  - Name: &par7 Position_f(0)(980)
+    Value: 0.99
+  - Name: &par8 Width_f(0)(980)
+    Value: 0.06
+  - Name: &par9 MesonRadius_f(0)(980)
+    Value: 1.0
+    Fix: true
+    Type: MesonRadius
```

Comment:
Is the `Type` actually used/needed?

Reply:
Nope 😬 Remove it?

Reply:
Yes. Probably we also don't need it for the mag phase Type. But there we might have to put a little bit of thought into it, as we want to support also real and imaginary part

Reply:
That's true. I ask, because the current implementation does not hurt: it's not used, but we're not sure if it's needed.

_Note_: types are note provided for the mass and with (pole position) parameters

Reply:
You can also leave it but we should open an issue then

Reply:
#161

Reply:
https://github.com/ComPWA/tensorwaves/issues/161

Reply:
I opened a new issue for this #397